jdaviz/configs/cubeviz/plugins/tools.py

```diff
@@ -2,28 +2,48 @@
 import os
 
 from glue.config import viewer_tool
+from glue_jupyter.bqplot.image import BqplotImageView
 from glue.viewers.common.tool import CheckableTool
 
 from jdaviz.configs.imviz.plugins.tools import _MatchedZoomMixin
 from jdaviz.core.events import SliceToolStateMessage
-from jdaviz.core.tools import PanZoom, SinglePixelRegion
+from jdaviz.core.tools import PanZoom, BoxZoom, SinglePixelRegion
 
 __all__ = []
 
 ICON_DIR = os.path.join(os.path.dirname(__file__), '..', '..', '..', 'data', 'icons')
 
 
+class _CubevizMatchedZoomMixin(_MatchedZoomMixin):
```

Comment:
Should the change be here too or no? Feels a little weird that the generically named PixelMatchPanZoom inherits from a mixin that specifically mentions Cubeviz. I cannot decide.
```suggestion
class _PixelMatchedZoomMixin(_MatchedZoomMixin):
```

Reply:
good idea - I'll make that change